chainstate/storage/src/lib.rs

```diff
@@ -81,10 +82,15 @@ pub trait BlockchainStorageRead: UtxosStorageRead {
 
     /// Get block tree as height vs ids
     fn get_block_tree_by_height(&self) -> crate::Result<BTreeMap<BlockHeight, Vec<Id<Block>>>>;
+
+    /// Get accounting undo for specific block
+    fn get_accounting_undo(&self, id: Id<Block>) -> crate::Result<Option<AccountingBlockUndo>>;
 }
 
 /// Modifying operations on persistent blockchain data
-pub trait BlockchainStorageWrite: BlockchainStorageRead + UtxosStorageWrite {
+pub trait BlockchainStorageWrite:
+    BlockchainStorageRead + UtxosStorageWrite + PoSAccountingStorageWrite
```

Comment:
Seems to me this should implement `PoSAccountingStorageRead` as well?

Reply:
`PoSAccountingStorageRead` is a supertrait of `PoSAccountingStorageWrite` so it's already there